invenio_app_ils/vocabularies/data/ill_cancel_reason.json

```diff
@@ -0,0 +1,23 @@
+[
+
+  {
+    "type": "ill_cancel_reason",
+    "key": "USER_CANCEL",
```

Comment:
I am not sure about these values... were they copied from the document request ones or provided by the library?

Reply:
They were copied from the document request ones, as I understood it from the [issue](https://github.com/CERNDocumentServer/cds-ils/issues/328).

Reply:
Ok I see, for me they do not make much sense in this case. For example, a user cannot cancel a borrowing request.

I would have something more generic here, and then override in CDS-ILS and ask our library what are the values that they need.

Reply:
Just for the record, librarians agreed to keep cancel_reason as a text input.